filter_test.go

```diff
@@ -242,3 +274,210 @@ func TestPolicyAssembleDefault(t *testing.T) {
 		}
 	}
 }
+
+func TestSimpleLongList(t *testing.T) {
+
+	syscallNumbers := make([]int, 0, len(arch.X86_64.SyscallNumbers))
+	for nr := range arch.X86_64.SyscallNumbers {
+		syscallNumbers = append(syscallNumbers, nr)
+	}
+	sort.Ints(syscallNumbers)
+
+	names := make([]string, 0)
```

Comment:
```suggestion
	names := make([]string, 0, 6)
```
(or just `var names []string` since we are in tests and efficiency doesn't matter — same where I've made similar comments below)